admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
----
-showToc: false
----
-<swagger>https://raw.githubusercontent.com/mojaloop/central-ledger/master/src/api/interface/swagger.json</swagger>
\ No newline at end of file
+---
+showToc: true
+---
+# Introduction
+
+This document provides detailed information about the Central Ledger API. The Central Ledger API is a Mojaloop API enabling Hub Operators to manage admin processes around:
+
+- creating/activating/deactivating participants in the Hub
+- adding and updating participant endpoint information
+- managing participant accounts, limits, and positions
+- creating Hub accounts
+- performing Funds In and Funds Out operations
+- creating/updating/viewing settlement models
+- retrieving transfer details
+
+ For background information about the participant and settlement model details that the Hub Operator can administer using the Central Ledger API, see section [Basic concepts](#basic-concepts).
+
+## Basic concepts
+
+To provide context for the admin operations that the Central Ledger API enables, this section gives a brief definition of some basic concepts.
+
+**Participant**
+Either the Hub itself or a Digital Financial Service Provider (DFSP) that is a participant in a Mojaloop scheme.
+
+**Endpoint**
+The DFSP callback URL where the Hub routes API callbacks. The URL specified is the endpoint set up in the outbound API gateway.
+
+**Limit**
+Also called "Net Debit Cap (NDC)". The NDC is a limit or a cap placed on a DFSP's funds available for transacting. It represents the maximum amount of money that the DFSP can owe to other DFSPs. The Position is continuously checked against the Net Debit Cap and when the Position amount exceeds the NDC amount, transfers are blocked.

```

Comment:
I would clarify that only the "NDC" is supported at this time. The idea of the Limits is to provide extensibility for future limit requirements.

Reply:
I might suggest a precision....

"...if a Transfer would cause the Position amount to exceed the NDC amount, the transfer is blocked."

I believe the formula, (TransferAmount + Position) <= NDC is checked dynamically, it doesn't cause a "bit to flip" and all future transfers to be blocked.

Reply:
OK, thank you, I have made the change.

Reply:
Perhaps it's easier to spot where the change is here: https://github.com/mojaloop/mojaloop-specification/pull/86/commits/ab5dfaea1627cf74b0115f3550b2b382dce71398